nixos/modules/services/misc/jellyfin.nix

```
@@ -24,6 +24,40 @@ in
         description = lib.mdDoc "Group under which jellyfin runs.";
       };
 
+      dataDir = mkOption {
+        type = types.path;
+        default = "/var/lib/jellyfin";
+        description = lib.mdDoc ''
+          base data directory, see https://jellyfin.org/docs/general/administration/configuration/#data-directory
+        '';
+      };
+
+      configDir = mkOption {
+        type = types.path;
+        default = "${cfg.dataDir}/config";
+        defaultText = "\${cfg.dataDir}/config";
```

```suggestion
        defaultText = "''${cfg.dataDir}/config";
```

The proper way to escape a nix expression in a string is `''${expression}`.

---

`''$` is an escape when the string is `'' ''`. For double quoted strings, `\$` is appropriate

---

Ah, okay then, I had also seen `''$` before for escaping in double quotes, so I was wondering. But okay, new thing learned :D